src/server/main_service.cc

```diff
@@ -2424,7 +2424,7 @@ void Service::Register(CommandRegistry* registry) {
   using CI = CommandId;
   registry->StartFamily();
   *registry
-      << CI{"QUIT", CO::READONLY | CO::FAST, 1, 0, 0, acl::kQuit}.HFUNC(Quit)
+      << CI{"QUIT", CO::FAST, 1, 0, 0, acl::kQuit}.HFUNC(Quit)
```

Comment:
the problem with this is that now QUIT might not work on replica.

Reply:
😅 either replication needs to handle this command as a special case or i can do that in client tracking to filter out this cid...

Reply:
it's my mistake. we only reject commands that have explicit `CO::WRITE` mask.